roles/pulp_common/tasks/install_packages.yml

```diff
@@ -48,6 +51,9 @@
             option: best
             value: "False"
             no_extra_spaces: true
+            mode: 0644
+            owner: root
+            group: root
```

Comment:
didn't we recently have a PR to avoid root as the owner?

Reply:
I ran `git log` but I do not see anything.